pennylane/devices/default_gaussian.py

```diff
@@ -572,7 +572,8 @@ def poly_quad_expectations(cov, mu, wires, device_wires, params, hbar=2.0):
 
     # HACK, we need access to the Poly instance in order to expand the matrix!
     # TODO: maybe we should make heisenberg_obs a class method or a static method to avoid this being a 'hack'?
-    op = qml.ops.PolyXP(Q, wires=wires)
+    # Linting check disabled as static analysis can misidentify qml.ops as the set instance qml.ops.qubit.ops
+    op = qml.ops.PolyXP(Q, wires=wires)  # pylint:disable=no-member
```

Comment:
Could we instead just call it `qml.PolyXP(Q, wires=wires)`? Then we wouldn't have to silence a warning.

Reply:
Good point!